bin/sync-groups-inventory.bash

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -eu -o pipefail
+
+
+here="$(dirname "$(readlink -f "$0")")"
```

Comment:
This is broken for me, and I'm thinking that given how well generate groups inventory works this feels redundant.

Reply:
It should work for a static inventory, I'll try it on a dynamic one

Reply:
The sync now works except for the `ansible_user` becomes null with `ansible_ssh_user`.

Reply:
Fixed the issues for the dynamic inventory, hopefully, it should work fine this time